backend/main/models/community.go

```diff
@@ -178,46 +184,39 @@ func (c *Community) CreateCommunity(db *s.Database) error {
 	)
 	RETURNING id, created_at
 	`, c.Name, c.Category, c.Logo, c.Slug, c.Strategies, c.Strategy, c.Banner_img_url, c.Website_url, c.Twitter_url, c.Github_url, c.Discord_url, c.Instagram_url, c.Terms_and_conditions_url, c.Proposal_validation, c.Proposal_threshold, c.Body, c.Cid, c.Creator_addr, c.Contract_name, c.Contract_addr, c.Public_path, c.Threshold, c.Only_authors_to_submit).Scan(&c.ID, &c.Created_at)
-	return err // will be nil unless something went wrong
+	return err
 }
 
-func (c *Community) UpdateCommunity(db *s.Database, payload *UpdateCommunityRequestPayload) error {
-	// First, merge the CommunityRequst
-	data, err := json.Marshal(payload)
-	if err != nil {
-		return err
-	}
-
-	json.Unmarshal(data, c)
-
-	_, err = db.Conn.Exec(
+func (c *Community) UpdateCommunity(db *s.Database, p *UpdateCommunityRequestPayload) error {
+	_, err := db.Conn.Exec(
 		db.Context,
 		`
 	UPDATE communities
 	SET name = $1, body = $2, logo = $3, strategies = $4, strategy = $5, 
 		banner_img_url = $6, website_url = $7, twitter_url = $8, github_url = $9,
-		discord_url = $10, instagram_url = $11, proposal_validation = $12, proposal_threshold = $13, category = $14, terms_and_conditions_url = $15
+		discord_url = $10, instagram_url = $11, proposal_validation = $12, 
+		proposal_threshold = $13, category = $14,
+		terms_and_conditions_url = $15
 	WHERE id = $16
 	`,
-		c.Name,
```

Comment:
is the entire community object submitted in the update payload, including fields that arent being updated?  just want to avoid the scenario where a user is just updating a single field, and all the other fields get nulled out.

Reply:
only the strategies array of object is submitted to the updateProposals endpoint. But all the fields within it that are not being updated are still being updated, so should not get nulled out. Tested this a bit with German. This is now the update payload, so here, `token-weighted-default` is added. But `one-address-one-vote` is still sent. Then we just update the whole json in that row. 

```
{
    "strategies": [
        {
            "name": "one-address-one-vote",
            "contract": {
                "name": "contactName",
                "addr": "0x11111111",
                "publicPath": "some_path",
                "threshold": "1.000",
                "maxWeight": "0.1"
            }
        },
        {
            "name": "token-weighted-default",
            "contract": {
                "name": "someotherNameContract",
                "addr": "0x111111",
                "publicPath": "some_path",
                    "threshold": "1.000",
                    "maxWeight": "0.1"
            }
        }
    ],
    "signingAddr": "0xf8d6e0586b0a20c7",
    "timestamp": "1655943178702",
    "compositeSignatures": [
        {
            "addr": "0xf8d6e0586b0a20c7",
            "keyId": 0,
            "signature": "7038617775902d7b4ae082e5d4284d29be6f76a2de92b54fab308063f6201f3ec485cfb4f9135dda83abfdae764fb225c7419f8ec7b4437fd7c86157a96bf3d6"
        }
    ]
}
```